tests/test_dialect_types.py

```diff
@@ -0,0 +1,41 @@
+import sqlalchemy_redshift.dialect
+import sqlalchemy
+
+def test_defined_types():
+    # These types are the officially supported Redshift types as of 20191121
```

Comment:
Can you add a link to Redshift docs page where this list is documented?

Reply:
Added!